verticapy/tests_new/core/vdataframe/test_plotting.py

```diff
@@ -0,0 +1,272 @@
+"""
+Copyright  (c)  2018-2024 Open Text  or  one  of its
+affiliates.  Licensed  under  the   Apache  License,
+Version 2.0 (the  "License"); You  may  not use this
+file except in compliance with the License.
+
+You may obtain a copy of the License at:
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless  required  by applicable  law or  agreed to in
+writing, software  distributed  under the  License is
+distributed on an  "AS IS" BASIS,  WITHOUT WARRANTIES
+OR CONDITIONS OF ANY KIND, either express or implied.
+See the  License for the specific  language governing
+permissions and limitations under the License.
+"""
+from itertools import chain
+
+import numpy as np
+import pytest
+
+import verticapy as vp
+
+
+class TestPlotting:
+    """
+    test class for Plotting functions test
+    """
+
+    def test_boxplot(self):
+        """
+        test function - boxplot
+        """
+        assert (
+            True
+        ), "This test is covered under /VerticaPy/verticapy/tests_new/plotting/plotly/test_plotly_boxplot.py"
+
+    def test_bar(self):
+        """
+        test function - bar
+        """
+        assert (
+            True
+        ), "This test is covered under /VerticaPy/verticapy/tests_new/plotting/plotly/test_plotly_bar.py"
+
+    def test_barh(self):
+        """
+        test function - barh
+        """
+        assert (
+            True
+        ), "This test is covered under /VerticaPy/verticapy/tests_new/plotting/plotly/test_plotly_barh.py"
+
+    def test_pie(self):
+        """
+        test function - pie
+        """
+        assert (
+            True
+        ), "This test is covered under /VerticaPy/verticapy/tests_new/plotting/base_test_files.py"
+
+    def test_hist(self):
+        """
+        test function - hist
+        """
+        assert (
+            True
+        ), "This test is covered under /VerticaPy/verticapy/tests_new/plotting/base_test_files.py"
+
+    def test_density(self):
+        """
+        test function - density
+        """
+        assert (
+            True
+        ), "This test is covered under /VerticaPy/verticapy/tests_new/plotting/base_test_files.py"
+
+    @pytest.mark.parametrize(
+        "start_date,end_date",
+        [
+            (None, None),
+            (1952, None),
+            (None, 1992),
+            # Fail - not considering start date when start and end dates are given
+            # (1952, 1992),
+        ],
+    )
+    @pytest.mark.parametrize(
+        "kind",
+        [
+            # Fail - vDataframe and vColumns results are non same
+            # "area_percent",
+            "area_stacked",
+            "line",
+            "spline",
+            "step",
+            None,
+        ],
+    )
+    @pytest.mark.parametrize("input_type", ["vDataframe", "vColumn"])
+    def test_plot(self, gapminder_vd_fun, input_type, start_date, end_date, kind):
```

Comment:
This test is also covered in tests_new/plotting/base_test_files.py 
VDCLinePlot, VDFLinePlot

Though I can see it only has following options: @pytest.mark.parametrize("kind", ["spline", "area", "step"])

You can augment that.

Reply:
Thanks. Have removed this test from verticapy/tests_new/core/vdataframe/test_plotting.py and added extra arguments to  tests_new/plotting/base_test_files.py
    @pytest.mark.parametrize("kind", ["area_percent", "area_stacked", "line", "spline", "step"])